easybuild/easyconfigs/g/GLib/GLib-2.44.1-intel-2015a-Python-2.7.10.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.44.1'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://ftp.gnome.org/pub/gnome/sources/glib/%(version_major_minor)s/']
+sources = ['glib-%(version)s.tar.xz']
+
+dependencies = [
+    ('libffi', '3.1'),
+    ('gettext', '0.19.2'),
+]
+python= 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s' %  (python, pyver)
```

Comment:
Well, the `GLib-2.44.1-intel-2015a.eb` already exist, with a Python-2.7.9 dependency. So I feel that I would break previous easyconfigs. I would refer here issue #2020 I did not have better idea to resolve this problem.